### DISCUSSION TITLE
欢迎试用腾讯云代码分析Jenkins插件

### DISCUSSION THREAD

**Discussion by zthzhang111** [Tencent/CodeAnalysis#149] [Announcements]
已经支持在Jenkins平台中通过插件使用腾讯云代码分析，使用方法参考：[TCA_Jenkins_Plugin.pdf](https://github.com/Tencent/CodeAnalysis/blob/main/plugins/jenkins_plugin/TCA_Jenkins_Plugin.pdf)

**Comment by mio4kon** [comment]
链接404了

**Reply by zthzhang111** [reply]
已修复，感谢您的反馈
